lib/puppetserver/ca/logger.rb

```diff
@@ -13,6 +13,10 @@ def initialize(level = :info, out = STDOUT, err = STDERR)
         @err = err
       end
 
+      def level
+        @level
+      end
+
```

Comment:
Instead of defining this method, you could use an `attr_reader`, or `attr_accessor` if you think it might be useful to set this value in addition to getting it. This has more info about those: https://www.rubyguides.com/2018/11/attr_accessor/ or feel free to chat with any of us about it!

Reply:
agree with Molly, to contrast, using a getter like this is valuable if you need to do some kind of translation, like if you wanted to return the symbolic name of the level.

Reply:
I did discuss that with Justin and I ended up going with a getter function. I could definitely make the change if it helps with readability.

Reply:
`attr_reader` is more of a Ruby specific idiomatic thing

Reply:
were you gonna make the change to use `attr_reader`?